mini-projects/movie-rec-system/movie_rec_system/recommendation/recommender.py

```diff
@@ -0,0 +1,77 @@
+import json
+import pandas as pd
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.metrics.pairwise import cosine_similarity
+from recommender_helper import (
+    content_movie_recommender,
+    get_popularity_rmse,
+    get_vote_avg_rmse,
+    get_vote_count_rmse,
+)
+
+
+def connect_duckdb():
+    """
+    Function that automatically connects
+    to duckdb as a GET call upon launch
+    of FastAPI
+
+    Returns a connection
+    """
+    # Will have to adjust this based on
+    # how we set up duckdb instance
+    pass
+
+
+def create_combined(df, weight=2):
```

Comment:
please add more context
df could be anything

add a docstring and a sample of what df looks like

Reply:
Docstring has been added: 
https://github.com/ploomber/sql/pull/100/files#diff-9b1926a7b439d1798b4654dd83de94bab22f71c9bea3f3db147cde8e9f062b85R30-R63